mongo/src/main/java/com/redhat/lightblue/mongo/crud/BasicDocFinder.java

```diff
@@ -97,50 +96,61 @@ public long find(CRUDOperationContext ctx,
             }
             int size = cursor.count();
```

Comment:
cursor has a size() so it's a little confusing to me that this variable is size even though it's not the cursor size, it's the cursor count, which are two different values. So maybe count instead of size would be clearer.